salt/syspaths.py

```diff
@@ -21,9 +18,17 @@
 from __future__ import absolute_import, print_function, unicode_literals
 import sys
 import os.path
+import logging
 
 __PLATFORM = sys.platform.lower()
-
+typo_warning = False
+log = logging.getLogger(__name__)
+svl = ('ROOT_DIR', 'CONFIG_DIR', 'CACHE_DIR', 'SOCK_DIR',  # salt variable list
```

Comment:
I think I'd prefer to call this `EXPECTED_VARIABLES`, since it shouldn't be changing. It's also more obvious what `EXPECTED_VARIABLES` means, than coming across `svl` somewhere in your code.

Reply:
I can change it to that if you'd like. but I had a comment at the end of the line stating what svl stood for.

Reply:
That's how I knew what it meant :)

The reason I prefer having it spelled out is that the other places have the full definition as well. And if you have to put a comment you may as well put it in code.